File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 3/ex2.c
#include <stdio.h>

main()
{
    printf("Tabuada de 0 a 9\n\n");

    //vars
    int i,j;

    for(i=0; i<=9; i++)
    {
        printf("\nTabuada do %i\n", i);
        for(j=0;j<=10;j++)
        {
            printf("%d x %i = %i\n", j, i, j*i);

        }

    }


}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 3/src/Ex2.java
//Importando Scanner
import java.util.Scanner;

public class Ex2 {

	public static void main(String[] args){
		
		//Instanicando Scanner
		Scanner input = new Scanner(System.in);
		
		//vars
		int num, i;
		
		for( num=0; num<=9; num++ )
		{
			System.out.println("\nTabuada do " + num);
			for( i=0; i<=10; i++ )
			{
				System.out.printf("%d * %d = %d\n", i, num, i*num);
			}
		}
		
	}
}
